apps/studio/src/examples/streetlights-kafka.yml

```diff
@@ -2,7 +2,14 @@ asyncapi: 3.0.0
 info:
   title: Streetlights Kafka API
   version: 1.0.0
-  description: "The Smartylighting Streetlights API allows you to remotely manage the city lights.\n\n### Check out its awesome features:\n\n* Turn a specific streetlight on/off \U0001F303\n* Dim a specific streetlight \U0001F60E\n* Receive real-time information about environmental lighting conditions \U0001F4C8\n"
+  description: |-
```

Comment:
it doesn't render correctly for me there should be another level of indentation for the text. please make sure it render correctly in deploy preview.

Reply:
Indent has been adjusted in streetlights, and files.state.ts has been updated as well.

I performed manual test (copy file/content from github webpage in my patch-1 branch, paste into studio). Then it renders OK. 
I do not know how to refresh the deploy preview for the later commits, and studio does not install to my raspberry pi (ARM instead of AMD architecture).  Would you have a suggestion how I can improve?

Reply:
> I do not know how to refresh the deploy preview for the later commits

it re-renders automatically.

> studio does not install to my raspberry pi (ARM instead of AMD architecture).

it runs ok on my M1 Mac. it is ARM as well. am I messing something? 🤔